package.json

```diff
@@ -29,19 +29,23 @@
   },
   "homepage": "https://github.com/bs-community/skinview2d#readme",
   "dependencies": {
+    "babel-loader": "^8.0.6",
```

Comment:
Is this intended?

Reply:
Yeah sorry. I already resolved the last one but did not make a comment until after, it's for storybook.

Reply:
I need to move it back to devDeps.